docs/UserGuide.md

```diff
@@ -218,9 +272,22 @@ Recipe data are saved as a JSON file `[JAR file location]/data/recipebook.json`.
 If your changes to the data file makes its format invalid, ***RIZZ***ipe will discard all data and start with an empty data file at the next run.
 </div>
 
-### Archiving data files `[coming in v2.0]`
+### Importing data
+
+The keyboard input `F3` will open an import window from the main window (as shown in the image below). From there, select the JSON
+file to be imported using the selector.
 
-_Details coming soon ..._
+* Only files with correct and non-duplicate recipes will be imported
```

Comment:
correct is a bit ambiguous, can be maybe "only files with recipe that pass our formatting checks"